Sources/MongoSwift/MongoCursor.swift

```diff
@@ -41,6 +41,10 @@ public class MongoCursor<T: Codable>: CursorProtocol {
     /// Decoder from the client, database, or collection that created this cursor.
     internal let decoder: BSONDecoder
 
+    /// The ID used by the server to track the cursor over time. If all of the cursor's results were returnable in a
+    /// batch, or if the cursor contained no results, this value will be nil.
+    public let id: Int64?
```

Comment:
as we discussed yesterday, this is now retrieved initially and stored rather than computing it

Reply:
One little nit: maybe say "a _single_ batch", or "_the first_ batch"? Seems just a tad clearer to me.

Reply:
woops, I meant to say that, this phrasing is definitely weird. fixed